ledger-bridge.js

```diff
@@ -140,11 +140,11 @@ export default class LedgerBridge {
         }
     }
 
-    async signTransaction (replyAction, hdPath, tx, to) {
+    async signTransaction (replyAction, hdPath, tx, to, chainId) {
         try {
             await this.makeApp()
-            if (to) {
-                const isKnownERC20Token = byContractAddress(to)
+            if (to && chainId) {
+                const isKnownERC20Token = byContractAddressAndChainId(to, chainId)
                 if (isKnownERC20Token) await this.app.provideERC20TokenInformation(isKnownERC20Token)
```

Comment:
Apparently we don't need to call this anymore, as of v5.52.0? https://github.com/LedgerHQ/ledgerjs/releases/tag/v5.52.0
I'm not 100% sure though.